model/src/user/flags.rs

```diff
@@ -6,19 +6,35 @@ use serde::{
 
 bitflags! {
     pub struct UserFlags: u64 {
-        const DISCORD_EMPLOYEE = 1;
-        const DISCORD_PARTNER = 1 << 1;
-        const HYPESQUAD_EVENTS = 1 << 2;
-        const BUG_HUNTER = 1 << 3;
-        const HOUSE_BRAVERY = 1 << 6;
-        const HOUSE_BRILLIANCE = 1 << 7;
-        const HOUSE_BALANCE = 1 << 8;
-        const EARLY_SUPPORTER = 1 << 9;
-        const TEAM_USER = 1 << 10;
+        /// Discord Employee.
+        const STAFF = 1;
+        /// Partnered server owner.
+        const PARTNER = 1 << 1;
+        /// HypeSquad events coordinator.
+        const HYPESQUAD = 1 << 2;
+        /// Bug hunter level 1.
+        const BUG_HUNTER_LEVEL_1 = 1 << 3;
+        /// House Bravery member.
+        const HYPESQUAD_ONLINE_HOUSE_1 = 1 << 6;
+        /// House Brilliance member.
+        const HYPESQUAD_ONLINE_HOUSE_2 = 1 << 7;
+        /// House Balance member.
+        const HYPESQUAD_ONLINE_HOUSE_3 = 1 << 8;
+        /// Early Nitro supporter.
+        const PREMIUM_EARLY_SUPPORTER = 1 << 9;
+        /// User is in a team.
+        const TEAM_PSEUDO_USER = 1 << 10;
+        /// Bug hunter level 2.
         const BUG_HUNTER_LEVEL_2 = 1 << 14;
+        /// Verified bot.
         const VERIFIED_BOT = 1 << 16;
-        const VERIFIED_BOT_DEVELOPER = 1 << 17;
-        const DISCORD_CERTIFIED_MODERATOR = 1 << 18;
+        /// Early verified bot developer.
+        const VERIFIED_DEVELOPER = 1 << 17;
+        /// Discord certified moderator.
+        const CERTIFIED_MODERATOR = 1 << 18;
+        /// Bot uses only HTTP interactions and is shown in the online member
+        /// list.
+        const BOT_HTTP_INTERACTIONS = 1 << 19;
```

Comment:
Can the old values stay there, but as deprecated? As far as I know there are no issues with having duplicate bitflag values.

Reply:
This should have been a request for changes